node/src/chain_spec.rs

```diff
@@ -68,23 +79,43 @@ pub fn roccocco_parachain_config(para_id: ParaId) -> Result<ChainSpec, String> {
 				wasm_binary,
 				// Sudo account
 				hex!["30035c21ba9eda780130f2029a80c3e962f56588bc04c36be95a225cb536fb55"].into(),
```

Comment:
What are these addresses? Don't we want alice/bob for testnet?

Reply:
Ah might have misunderstood the use of this config. What's the purpose? Real testing?

Reply:
yes, this is config for public/real testing(westend/rococo).